SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -10,13 +10,42 @@
 #include "Async/Async.h"
 #include "Improbable/SpatialEngineConstants.h"
 #include "Improbable/SpatialGDKSettingsBridge.h"
+#include "Interfaces/IPluginManager.h"
 #include "Misc/Paths.h"
 #include "Modules/ModuleManager.h"
 
 DEFINE_LOG_CATEGORY(LogSpatialConnectionManager);
 
 using namespace SpatialGDK;
 
+class GDKVersionLoader
+{
+public:
+	static FString GetGDKVersion()
+	{
+		if (GDKVerson.IsEmpty())
+		{
+			IPluginManager& PluginManager = IPluginManager::Get();
+			TArray<TSharedRef<IPlugin>> Plugins = PluginManager.GetDiscoveredPlugins();
+			for (const TSharedRef<IPlugin>& Plugin : Plugins)
```

Comment:
Hmmm, we really have to iterate over each plugin to get the one we want :o?
Does `IPluginManager::FindPlugin` work?

Reply:
Yes that will work... I must have missed this in that API...